<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RentCarWindow</class>
 <widget class="QDialog" name="RentCarWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rent Car</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/res/data/icons/add.png</normaloff>:/res/data/icons/add.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#frameRent
{
        background-color:#333;
}

QLabel#RentCarlbl
{
        font-size:20px;
        background-color:#222;
        border-radius:12px;
}

QLabel#NumOfDays, QLabel#PickStartDay
{
        font-size:11px;
        background-color:#222;
        border-radius:12px;
}

QCalendarWidget#calendarWidget
{
	  color: rgb(180, 180, 180);
      background-color: #111;
      selection-background-color: rgb(233, 60, 44 );
      selection-color: rgb(255, 255, 255 );
          font-size:10px;
alternate-background-color: rgb(128, 128, 128);
}
QSpinBox#spinBox
{
        background-color:#222;
font-size:11px;
}

QCalendarWidget#calendarWidget
{
         color: rgb(180, 180, 180);
      background-color: black;
      selection-background-color: rgb(233, 60, 44 );
      selection-color: rgb(255, 255, 255 );
}
QPushButton#RentRequ
{
        background-color:#222;
        font-size:11px;
}

QPushButton#RentRequ:hover
{
        background-color:#444;
        font-size:11px
}

QLabel#CalWarning,QLabel#label_3
{
        font-size:11px;
color:red;
}
QLabel#payment
{
        font-size:11px;
        background-color:#222;
        border-radius:12px;
}
QLineEdit,QRadioButton{
font-size:11px;
}</string>
  </property>
  <widget class="QLabel" name="PickStartDay">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> Pick a starting day :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RentRequ">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>520</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Request Rental</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameRent">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>431</width>
     <height>591</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="RentCarlbl">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>   Rent Car :</string>
    </property>
   </widget>
   <widget class="QLabel" name="NumOfDays">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Pick the number of Days you wish to rent the car:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="CalWarning">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="cash">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Cash</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="visa">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Visa</string>
    </property>
   </widget>
   <widget class="QLabel" name="card">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>420</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#card
{
        font-size:11px;
        background-color:#222;
        border-radius:12px;
}</string>
    </property>
    <property name="text">
     <string>Card number</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pin">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>450</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#pin
{
        font-size:11px;
        background-color:#222;
        border-radius:12px;
}</string>
    </property>
    <property name="text">
     <string>PIN</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>420</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>1234 5678 1234 5678</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>450</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>123</string>
    </property>
   </widget>
   <widget class="QLabel" name="payment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Payment method:</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>280</width>
      <height>156</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	  color: rgb(180, 180, 180);
      background-color: #111;
      selection-background-color: rgb(233, 60, 44 );
      selection-color: rgb(255, 255, 255 );
          font-size:10px;
alternate-background-color: rgb(128, 128, 128);</string>
    </property>
   </widget>
  </widget>
  <zorder>frameRent</zorder>
  <zorder>PickStartDay</zorder>
  <zorder>RentRequ</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
